openshift/rh-che.app.yaml

```diff
@@ -263,6 +263,16 @@ objects:
               secretKeyRef:
                   key: che.jdbc.username
                   name: che
+          - name: CHE_AUTH_SERVICE_ACCOUNT_SECRET
```

Comment:
I believe that these names don't comply with Che env vars naming policy. Consider 
CHE_OPENSHIFT_SERVICE__ACCOUNT__ID
CHE_OPENSHIFT_SERVICE__ACCOUNT__SECRET

Reply:
Or 
CHE_OPENSHIFT_SERVICE__ACCOUNT_ID
CHE_OPENSHIFT_SERVICE__ACCOUNT_SECRET